api/server/runner.go

```diff
@@ -164,7 +165,7 @@ func (s *Server) serve(ctx context.Context, c *gin.Context, appName string, foun
 	envVars := map[string]string{
 		"METHOD":      c.Request.Method,
 		"ROUTE":       found.Path,
-		"REQUEST_URL": c.Request.URL.String(),
+		"REQUEST_URL": fmt.Sprintf("%v%v", viper.GetString(EnvAPIURL), c.Request.URL.String()),
```

Comment:
Using the `API_URL` in this change is not the correct URL. But why is `c.Request.URL.String()` not returning the full URL?  Might have to use Request.Host or Request.RequestURI or something.